Dockerfile

```diff
@@ -1,10 +1,11 @@
-# BUILD IMAGE ------------------------------------------------------------------
+# BUILD NIM APP ----------------------------------------------------------------
 
 FROM alpine:edge AS nim-build
 
 ARG NIMFLAGS
 ARG MAKE_TARGET=wakunode2
 ARG EXPERIMENTAL=false
+ARG NIM_COMMIT_ARG
```

Comment:
I don't think the ARG shoul have the suffix ARG:)

Reply:
ah yes, I change it now. thanks :)